otelcollector/fluent-bit/src/Makefile

```diff
@@ -0,0 +1,22 @@
+BASE_DIR := $(subst /build/linux,,$(PWD))

```

Comment:
does this make file gets picked up & built when we do make from otelcollector folder ?

Reply:
I have added this in now